asv_bench/benchmarks/benchmarks.py

```diff
@@ -29,43 +29,72 @@
     random_columns,
     random_booleans,
 )
-from modin.config import NPartitions
+
+try:
+    from modin import config  # noqa: F401
+
+    OLD_MODIN_VERSION = False
+except ImportError:
+    OLD_MODIN_VERSION = True
+
+try:
+    from modin.config import NPartitions
+
+    NPARTITIONS = NPartitions.get()
+except ImportError:
+    if not OLD_MODIN_VERSION:
+        from modin.config import CpuCount
+
+        NPARTITIONS = CpuCount.get()
+    else:
+        import os
+        import multiprocessing
+
+        NPARTITIONS = int(os.environ.get("MODIN_CPUS", multiprocessing.cpu_count()))
```

Comment:
in a previous version number of partitions was stored in `pd.DEFAULT_NPARTITIONS`
```suggestion
except ImportError:
    NPARTITIONS = pd.DEFAULT_NPARTITIONS
```